.github/workflows/04_release_deploy.yml

```diff
@@ -64,6 +64,12 @@ jobs:
       semver: ${{ steps.get_semver.outputs.semver }}
       environment: ${{ steps.output.outputs.environment }}
     steps:
+      - name: Semver setup
+        id: semver_setup
+        uses: pagopa/github-actions-template/nodo5-semver-setup@ce252c8501c9242bd6045f7cdd650736b2f38777
+        with:
+          semver: ${{ inputs.semver }}
+
       # Check branch or tag
       - name: Checkout code
```

Comment:
in order to add `Semver setup` step, you must remove the other following steps, as done [here](https://github.com/pagopa/pagopa-fdr-json-to-xml/blob/bca008638cbb3802d76573843fa18145f118edbf/.github/workflows/04_release_deploy.yml#L61)